tempesta_fw/cache.c

```diff
@@ -1571,6 +1572,23 @@ tfw_cache_build_resp(TfwCacheEntry *ce)
 	return NULL;
 }
 
+static inline bool
+tfw_cache_set_hdr_age(TfwHttpMsg *hmresp, TfwCacheEntry *ce)
+{
+	size_t digs;
+	char cstr_age[TFW_ULTOA_BUF_SIZ] = {0};
+	time_t age = tfw_cache_entry_age(ce);
+
+	if (!(digs = tfw_ultoa(age, cstr_age, TFW_ULTOA_BUF_SIZ)))
+		return false;
+
+	if (tfw_http_msg_hdr_xfrm(hmresp, "Age", sizeof("Age") - 1,
+				  cstr_age, digs, TFW_HTTP_HDR_RAW, 0))
+		return false;
+
+	return true;
```

Comment:
We have many same functions that add or replace one header, all of them return error code, not bool value. See `tfw_http_adjust_req()` for example. All the functions have the same logic,  so the return value  should be the same.